inc/commondbtm.class.php

```diff
@@ -166,13 +178,13 @@ function __construct () {
     * Get known tables
     *
     * @return array
+    *
+    * @deprecated 9.4.2
     */
    public static function getTablesOf() {
-      global $GLPI_CACHE;
-      if ($GLPI_CACHE != null && $GLPI_CACHE->has('table_of')) {
-         return $GLPI_CACHE->get('table_of');
-      }
-      return [];
+      Toolbox::deprecated();
```

Comment:
Guess this method can be kept and return `self::$tables_of` (if this is already used, in some plugins for example). If not used, ok to deprecate :)

Reply:
I do not see any usage in plugins, and it returns the computed value so it will just trigger a deprecation but will not fail.